contrib/examples/actions/echochain.meta.yaml

```diff
@@ -10,4 +10,11 @@ runner_type: "action-chain"
 entry_point: "chains/echochain.yaml"
 
 enabled: true
-parameters: {}
+parameters:
+  skip_notify:
```

Comment:
Edit: I see, it's a string.

Any reason why it's a string and not a list (array)?